config/default/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: awx-resource-operator-system
+namespace: ansible-automation-platform
```

Comment:
For upstream, can we make this awx, or awx-resource?

I can add logic midstream to convert this to `aap` for downstream (starting in 2.3, the default namespace is aap)